### ISSUE TITLE
Youtube still showing advert suddently but previously did not

### ISSUE THREAD

**Issue by tomloaf** [uBlockOrigin/uBlock-issues#2788]
### Prerequisites

- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] using a new, unmodified browser profile.

### Description

Youtube still showing advert suddently but previously did not

### A specific URL where the issue occurs.

https://www.youtube.com/watch?v=v5A92I_LdU4

### Steps to Reproduce

Open any youtube video and start to play advert when it should not

### Expected behavior

not play advert

### Actual behavior

play advert with no image

### Configuration

.

**Comment by aaron-watts** [comment]
This may only be an issue on chrome browsers and not firefox. I encountered this on ungoogled chromium but when I checked librewolf the yt ads were blocked.

**Comment by twilight-sparkle-irl** [comment]
Having this issue on latest Firefox.

**Comment by tomloaf** [comment]
Hi

I am using firefox

**Comment by stephenhawk8054** [comment]
PLEASE just continue here: https://github.com/uBlockOrigin/uAssets/issues/7636

This is not the place to report filter lists issues.

**Comment by Spirits-** [comment]
I don't know if this is a filter lists issue, as reloading the youtube video gets rid of the preroll ads while they are rolling.

Chromium based browser (Vivaldi 6.1.3035.302 (Stable channel) (64-Bit)), uBlock 1.51.0

**Comment by stephenhawk8054** [comment]
@Spirits- Yes, it's filter lists issue, the filter lists cover 2 cases: when opening from fresh URL (or reloading) AND when browsing youtube. The current issue is for the latter.
